src/liteclient.cc

```diff
@@ -243,18 +243,19 @@ bool LiteClient::finalizeInstall(data::InstallationResult* ir) {
   return ret.result_code.num_code == data::ResultCode::Numeric::kOk;
 }
 
-Uptane::Target LiteClient::getRollbackTarget() {
-  const auto rollback_hash{sysroot_->getDeploymentHash(OSTree::Sysroot::Deployment::kRollback)};
+Uptane::Target LiteClient::getRollbackTarget(bool ignore_ostree_hash) {
+  const auto rollback_hash{ignore_ostree_hash ? ""
+                                              : sysroot_->getDeploymentHash(OSTree::Sysroot::Deployment::kRollback)};
 
   Uptane::Target rollback_target{Uptane::Target::Unknown()};
   {
     std::vector<Uptane::Target> installed_versions;
     storage->loadPrimaryInstallationLog(&installed_versions,
-                                        true /* make sure that Target has been successfully installed */);
+                                        true /* make sure that Target has been successfully installed */, false);
```

Comment:
The same question about considering the current target as the target to rollback to under certain circumstances.

Reply:
A change here is probably needed, to allow current depending if the latest install has finished or not. I'll adjust it tomorrow.